package.json

```diff
@@ -188,5 +188,6 @@
       "node node_modules/prettier/bin/prettier.cjs --cache --write --ignore-unknown",
       "cspell --cache --no-must-find-files"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Let's remove it

Reply:
Yeah don't worry. I'll diff at the end and correct all the nonsense. My work machine has corepack enabled and it's difficult to enable disable between repos. 

Don't worry the refined PR will remove all the unrelated crap. This is intermediary to allow me to just focus on getting the work done. There will be a clean up pass so you only need to be concerned with the high level goal of the PR at this time. When it's not draft pr, you can be more thorough. 

There will probably be debugger and logs and lint disable throughout it till it's no longer draft stage.